CODEOWNERS

```diff
@@ -1,4 +1,5 @@
 # SPDX-License-Identifier: Apache-2.0
 
-# FireFly Core Maintainers
-* @awrichar @peterbroadhurst @nguyer
+# FireFly Helm Chart Maintainers
+* @drewmarshburn @peterbroadhurst @hfuss
```

Comment:
@nguyer let me know what process we'll need to go through to get this approved.

Reply:
For others reference @hfuss is the principal author of all of this helm chart to date (so obviously needs to be in the maintainer list).
@drewmarshburn has been using and extending the charts to deploy in multiple customer environments. He's not currently in the Hyperledger org, so that might need @nguyer's support, but I strongly support his addition to the maintainer list.

Reply:
Thanks @peterbroadhurst! Just chatted with @drewmarshburn and he'll work on joining the Discord and the GitHub org today.

Reply:
The current list of maintainers can be found here: https://github.com/orgs/hyperledger/teams/firefly-committers/members. Neither @hfuss nor @drewmarshburn are on that list. We would need to go through the process of adding them as maintainers before they could become CODEOWNERS for a specific repo. The process for that is here: https://github.com/hyperledger/firefly/blob/main/MAINTAINERS.md#becoming-a-maintainer

Reply:
Requested to join the maintainers list and will update the MAINTAINERS.md to reflect it for this specific repo. Thank you for the time and help with all this!

Reply:
Reverted this change for now so that this PR isn't blocked. We can edit this later once Drew and I are done with the nomination process.